common4j/src/main/com/microsoft/identity/common/java/commands/AcquirePrtSsoTokenResult.java

```diff
@@ -0,0 +1,56 @@
+package com.microsoft.identity.common.java.commands;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Map;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.experimental.Accessors;
+
+/**
+ * A DTO for the results from an AcquirePrtSsoToken request.
+ */
+@Builder
+@Getter
+@Accessors(prefix = "m")
+public class AcquirePrtSsoTokenResult {
+
+    /**
+     * Object holding auth error information
+     *
+     */
+    @SerializedName("error")
+    private final @Nullable String mError;
+    /**
+     * Account name.
+     *
+     */
+    @SerializedName("account")
+    private final @Nullable String mAccountName;
```

Comment:
I think we need to send the `IAccountRecord` back here

Reply:
at least I need to send back the account object in Linux

Reply:
Hm.  I was hoping not have to send all of that, but if it isn't something that can be understood from the name I can add it.

Reply:
The sending system should have the account already - this call is silent, right?

Reply:
Yes, it should. I can probably put the same account object back on it.